public/js/config.js

```diff
@@ -118,6 +120,11 @@ Config.enable_resize = function () {
     else $(".resizefields").hide();
 };
 
+Config.enable_localprogress = function () {
+    if ($("#localprogress").prop("checked")) $(".serversideprogressfields").hide();
+    else $(".serversideprogressfields").show();
+};
```

Comment:
Would it be better to have both options work at the same time? If both are enabled use server progress for authenticated requests and local progress for everything else.